filebeat/config/config.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // Defaults for config variables which are not set
 const (
-	DefaultRegistryFile                      = ".filebeat"
+	DefaultRegistryFile                      = "registry"
```

Comment:
Removed my +1 as I think the point about implementation and potentially changing in the future is a good point. So even if we switch to the format "xyz" the file name stays the same.